app/code/Magento/Customer/etc/webapi.xml

```diff
@@ -128,7 +128,13 @@
             <resource ref="anonymous"/>
         </resources>
     </route>
-    <route url="/V1/customers/:customerId" method="PUT">
```

Comment:
Unfortunately, we can't change webapi endpoint according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
@sidolov  I don't see how this breaks backward compatibility, I am not removing the PUT method I am just adding a POST one

Reply:
@judaschwartz But you changed `<route url="/V1/customers/:customerId" method="PUT">` to `<route url="/V1/customers/:id" method="PUT">`. I think is what @sidolov means.

Reply:
@torhoehn that change is actually a revert to how it was in 2.0 before someone changed it and broke this functionality https://github.com/magento/magento2/blob/2.0/app/code/Magento/Customer/etc/webapi.xml#L125